### DISCUSSION TITLE
Сортировка треков по альбому

### DISCUSSION THREAD

**Discussion by sewyonov** [Chimildic/YaMuTools#31] [Идея] [answered]
**Сейчас есть:**
По треку (А-Я)
По треку (Я-А)
По исполнителю (А-Я)
По исполнителю (Я-А)
По длительности (короткие - длинные)
По длительности (длинные короткие)
Случайно

**Предлагаю:**
По альбому (А-Я)
По альбому (Я-А)

**Comment by Chimildic** [comment, marked as answer]
Добавлено в версии 0.8.0
